nestkernel/model_manager.cpp

```diff
@@ -267,26 +269,16 @@ ModelManager::copy_connection_model_( index old_id, Name new_name )
   if ( new_id == invalid_synindex ) // we wrapped around (=63), maximal id of
                                     // connection_model = 62, see nest_types.h
```

Comment:
```suggestion
  if ( new_id == invalid_synindex )
```
Values in comment are outdated and depend on configuration, so I suggest to drop.